geometry_benchmark_test.go

```diff
@@ -35,32 +35,32 @@ func BenchmarkSize_Subtract(b *testing.B) {
 // because this slows down the speed by factor 10.
 func TestPosition_Add_Speed(t *testing.T) {
```

Comment:
Would it not make more sense to make these into actual benchmarks (which are turned off by default) instead of adding a build tag for disabling them?

Reply:
I’m undecided on this one. I think it is out-of-scope for this PR but probably would not veto a new PR changing this.

Reply:
Alright. I don't necessarily agree that it is out of scope but I will approve and open a PR for that.